parachain/utils/test-parachain/runtime/src/lib.rs

```diff
@@ -204,6 +208,11 @@ pub const UNIT: Balance = 1_000_000_000_000;
 pub const MILLIUNIT: Balance = 1_000_000_000;
 pub const MICROUNIT: Balance = 1_000_000;
 
+// The asset id for SnowEth
+pub const SNOW_ETH_ASSET_ID: u32 = 1000_0001;
```

Comment:
Can just use `1` as the asset id. Simplifies things a bit for clients that need to query the asset.

```suggestion
pub const SNOW_ETH_ASSET_ID: u32 = 1;
```